src/aat/resources/features/F-063/F-063_Base_Test_Data.td.json

```diff
@@ -0,0 +1,26 @@
+{
+	"_guid_": "F-063_Base_Test_Data",
+
+	"productName": "CCD Data Store",
+	"operationName": "Retrieve search input details for dynamic display",
+
+	"method": "GET",
+	"uri": "/internal/case-types/{ctid}/search-inputs",
+
+	"users": {
+    "invokingUser": {
+      "username": "[[$CCD_CASEWORKER_AUTOTEST_EMAIL]]",
+      "password": "[[$CCD_CASEWORKER_AUTOTEST_PASSWORD]]"
+    }
+	},
+
+	"request": {
+		"headers": {
+			"_extends_": "Common_Request_Headers",
```

Comment:
Do we need the following header with the endpoint?
public static final String UI_SEARCH_INPUT_DETAILS =
            "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-search-input-details.v2+json;charset=UTF-8";

Reply:
I believe we are checking the following endpoint.

		path = "/case-types/{caseTypeId}/search-inputs",

Reply:
it is included as a response header in S-215.td.json as it is only relevant for the happy path scenario

Reply:
okay....